crates/common/Cargo.toml

```diff
@@ -10,5 +10,7 @@ rust-version.workspace = true
 
 [dependencies]
 anyhow = { workspace = true }
+moka = { workspace = true, features = ["sync"] }
+once_cell = { workspace = true }
```

Comment:
This is already in std I believe (https://doc.rust-lang.org/std/cell/struct.LazyCell.html) or (https://doc.rust-lang.org/std/sync/struct.LazyLock.html) :)